src/Compiler/CompilerOptions.cpp

```diff
@@ -281,6 +282,19 @@ static llvm::cl::opt<std::string, true> shapeInformationOpt("shapeInformation",
     llvm::cl::value_desc("value"), llvm::cl::location(shapeInformation),
     llvm::cl::cat(OnnxMlirOptions));
 
+static llvm::cl::opt<std::string, true> dimParamsOpt("dimParams",
+    llvm::cl::desc(
+        "Custom onnx.dim_params attributes for the inputs of the ONNX model for"
+        "specifying relationship among ranks of the inputs.\n"
+        "\"value\" is in the format of "
+        "\"INPUT_ID1:D1=S1,D2=S2,...,Dn=Sn|INPUT_ID2:D1=T1,D2=T2,...Dn=Tn\""
```

Comment:
missing a `,` before that last `Dn`.

It's better to add `| ...` at the end to denote multiple INPUT_ID, e.g. 
```
INPUT_ID1:D1=S1,D2=S2,...,Dn=Sn | INPUT_ID2:D1=T1,D2=T2,...,Dn=Tn | ...
```

Reply:
Fixed. Thanks.